shared/charts/locust/values.yaml

```diff
@@ -38,3 +38,7 @@ worker:
     requests:
       cpu: 100m
       memory: 128Mi
+
+target:
+  labels:
+  namespace: gpii
```

Comment:
I think this line should be indented one more level.

Reply:
Done.